app/views/bill_details/index.html.erb

```diff
@@ -30,7 +30,7 @@
         </tbody>
     </table>
     <% if @bill_details.present? %>
-      <%= link_to t(".payment"), "#", class: "btn btn-success" %>
+      <%= link_to t(".payment"), payment_path(id: params[:bill_id]), class: "btn btn-success" %>
```

Comment:
check trường hợp params[:bill_id] nil

Reply:
Em check @bill_details.present? thì ms có hiển thị nút thanh toán.
Nên params[:bill_id] sẽ luôn tồn tại.